grabbit/src/main/groovy/com/twcable/grabbit/spring/batch/repository/GrabbitJobExecution.groovy

```diff
@@ -0,0 +1,28 @@
+package com.twcable.grabbit.spring.batch.repository
+
+import groovy.transform.CompileStatic
+import groovy.transform.InheritConstructors
+import org.springframework.batch.core.JobExecution
+
+/*
+ * Copyright 2015 Time Warner Cable, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@CompileStatic
+@InheritConstructors
+class GrabbitJobExecution extends JobExecution {
```

Comment:
Javadoc?  Specifically, what is this used for? Why is its sole property "transactionID"?